packages/@react-spectrum/picker/src/Picker.tsx

```diff
@@ -181,10 +181,11 @@ function Picker<T extends object>(props: SpectrumPickerProps<T>, ref: DOMRef<HTM
     );
   }
 
+  let inputText = state.selectedItem ? state.selectedItem.textValue : '';
```

Comment:
lets call this variable `textValue` if it doesn't conflict with anything else so that it's a bit easier to follow from here to down below